src/components/Search/Search.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import SearchButton from './SearchButton/SearchButton';
+import SearchInput from './SearchInput/SearchInput';
+
+import './Search.css';
+
+interface SearchProp {
+  searchQuery: string;
+  setSearchQuery: (value: string) => void;
+  setIsSubmitted: (value:boolean) => void;
+}
+
+const Search = (props: SearchProp) => {
+  const { searchQuery, setSearchQuery, setIsSubmitted } = props;
+  function handleSubmit(e: React.SyntheticEvent) {
```

Comment:
SyntheticEvent is a general type, be more specific and you use React.FormEvent<>